images/rook/Dockerfile.base

```diff
@@ -14,48 +14,32 @@
 
 FROM BASEIMAGE
 
-ARG EXTRALIBS
-
-# install runtime dependencies for ceph and rook
-RUN BOOST_VERSION=1.62.0 && \
+# add Ceph and other packages needed
+ADD 'https://download.ceph.com/keys/release.asc' /tmp/release.asc
+RUN CEPH_VERSION=luminous && \
+    apt-key add /tmp/release.asc && \
+    echo "deb http://download.ceph.com/debian-$CEPH_VERSION/ xenial main" | tee /etc/apt/sources.list.d/ceph-$CEPH_VERSION.list && \
     DEBIAN_FRONTEND=noninteractive apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -yy -q --no-install-recommends \
         ca-certificates \
-        gdisk \
+        ceph-common \
```

Comment:
if we're pulling the same binaries into the image, where does the increase in the image size come from? Is it an artifact of `apt-get`?

Reply:
The debian packages have more dependencies than actually needed for our scenarios

Reply:
@bassam I'd be interested to know what these are. Maybe it makes sense to slim them down in the Debian packaging.

Reply:
@ktdreyer [this](https://github.com/rook/rook/blob/master/images/ceph/Dockerfile#L26) is the list of packages needed to run the ceph daemons. I've not traced through the differences between this and the ones installed on `apt-get install ceph-common` but I've noticed its a much larger list.

IMO its well worth the effort to refactor the deb/rpms in upstream ceph to optimize for image size for running the daemons.

Reply:
Thanks